oca_dependencies.txt

```diff
@@ -1 +1,2 @@
 product-attribute
+stock-logistics-warehouse https://github.com/ForgeFlow/stock-logistics-warehouse.git 13.0-mig-stock_inventory_discrepancy
```

Comment:
Rebase and drop this line as https://github.com/OCA/stock-logistics-warehouse/pull/824 has already merged.